platform/backend/repository/sqlite/src/main/kotlin/io/hamal/repository/sqlite/record/code/Repository.kt

```diff
@@ -66,12 +67,14 @@ class SqliteCodeRepository(
                         cmdId = cmdId,
                         entityId = codeId,
                         groupId = cmd.groupId,
-                        value = cmd.value
+                        value = cmd.value,
+                        type = CodeType.Lua54
```

Comment:
```suggestion
                        type = cmd.type
```

lets do the hardcoding on the upper level